Content/build.fsx

```diff
@@ -137,17 +185,46 @@ Target.create "Bundle" (fun _ ->
 
 let dockerUser = "safe-template"
 let dockerImageName = "safe-template"
+//#endif
+//#if (deploy == "docker")
 let dockerFullName = sprintf "%s/%s" dockerUser dockerImageName
 
 Target.create "Docker" (fun _ ->
-    let buildArgs = sprintf "build -t %s ." dockerFullName
-    runTool "docker" buildArgs "."
+    buildDocker dockerFullName
+)
+
+//#endif
+//#if (deploy == "gcp-kubernetes")
+Target.create "Docker" (fun _ ->
+    let dockerTag = createDockerTag dockerImageName
+    buildDocker dockerTag
+)
+
+Target.create "Publish" (fun _ ->
+    let dockerTag = createDockerTag dockerImageName
+    let pushArgs = sprintf "push %s" dockerTag
+    runTool "docker" pushArgs "."
+)
 
-    let tagArgs = sprintf "tag %s %s" dockerFullName dockerFullName
-    runTool "docker" tagArgs "."
+Target.create "ClusterAuth" (fun _ ->
+    let clusterName = Environment.environVarOrDefault "SAFE_CLUSTER" "safe-cluster"
+    let authArgs = sprintf "container clusters get-credentials %s" clusterName
+    runTool "gcloud" authArgs "."
```

Comment:
`ERROR: (gcloud.container.clusters.get-credentials) One of [--zone, --region] must be supplied: Please specify location.`

Reply:
I've updated the docs to say how you set a default zone, with that set it should work. Is that enough?

Reply:
yep, fair enough!